CMakeLists.txt

```diff
@@ -700,4 +736,4 @@ foreach(option IN LISTS options)
     find_package_message("${option}" "${option}: ${value}" "${value}")
 endforeach(option)
 list_optional_packages()
-message (STATUS "")
+message (STATUS "")
```

Comment:
I added a check to checkpatch to complain about missing EOL at EOF. Please fix.

Reply:
Fixed: returned all EOLs at EOF.